Build.mak

```diff
@@ -19,6 +19,8 @@ $O/libdyaml.a: $(shell find $C/submodules/d-yaml/source -type f -name '*.d') \
 
 ################################################################################
 
+$O/allunittests $O/fastunittest: $O/libvibed.a $O/libdyaml.a
```

Comment:
You forgot to remove this line:
```make
%unittests: $O/libvibed.a $O/libdyaml.a
```

Reply:
Well, it is removed later. Do you want that removal to be moved to another commit?